src/main/scala/dpla/ingestion3/entries/ingest/IngestRemap.scala

```diff
@@ -0,0 +1,73 @@
+package dpla.ingestion3.entries.ingest
+
+import dpla.ingestion3.confs.{CmdArgs, Ingestion3Conf}
+import dpla.ingestion3.entries.ingest.EnrichEntry.executeEnrichment
+import dpla.ingestion3.entries.ingest.JsonlEntry.executeJsonL
+import dpla.ingestion3.entries.ingest.MappingEntry.executeMapping
+import dpla.ingestion3.entries.reports.ReporterMain.executeAllReports
+import dpla.ingestion3.utils.Utils
+import org.apache.spark.SparkConf
+
+/**
+  * Single entry point to run harvest, mapping, enrichment, indexing jobs and all reports.
+  * Support running against an already harvested data set and running a fresh harvest
+  *
+  * Expects four parameters:
+  *   1)  --output  A base path to save the mapping, enrichment, json-l and report outputs
+  *   2)  --input   A path to the harvested data
+  *   3)  --conf    A path to the application configuration file
+  *   4)  --name    Provider short name
+  */
+object IngestRemap {
+
+  def main(args: Array[String]): Unit = {
+
+    // Read in command line args.
+    val cmdArgs = new CmdArgs(args)
+
+    val baseDataOut = cmdArgs.getOutput()
+    val confFile = cmdArgs.getConfigFile()
+    val shortName = cmdArgs.getProviderName()
+
+    // Outputs
+    val harvestDataOut = cmdArgs.getInput()
```

Comment:
Wouldn't the above get `getOutput` instead of `getInput`?

Reply:
No, b/c this entry point expects harvest data already the "input" on the command line is the harvest data and the "output" is a base directory where we will put the output from mapping, enrichment, reports and jsonl.

base directory + "/mapping"
base directory + "/enrichment"
base directory + "/reports"
base directory + "/jsonl"

Reply:
Oh I see.  Why wouldn't you want this Object to harvest the data?  What's the motivation for keeping that part of the workflow separate?

Reply:
My goal with this was to have an entry point that is going to allow us to plow through CQA processes as we start to migrate providers and since harvest is the most expensive operation I thought it made sense to omit it here. 

I did have an earlier version where running a harvest was optional but since the letter of the ticket was map, enrich, index, report I dropped it off.  There should be another entry point for production that omits the report generation so this shouldn't be seem as the only execution point for us. Just one designed for testing mappings.

Reply:
That makes sense.  Thanks!